pkg/util/exec/fake_exec.go

```diff
@@ -143,3 +145,69 @@ func (fake *FakeExitError) Exited() bool {
 func (fake *FakeExitError) ExitStatus() int {
 	return fake.Status
 }
+
+func commandAsStringSlice(command interface{}) []string {
+	if args, ok := command.([]string); ok {
+		return args
+	} else if commandstr, ok := command.(string); ok {
+		return strings.Split(commandstr, " ")
```

Comment:
I guess that's a documentation issue: the idea was that you can pass a string when the command is going to be simple (ie, no spaces in arguments), and a []string otherwise. We can just demand []string, yes, it just makes it a little more annoying to use in the 99% of cases where you don't have args-with-spaces in them.

Note that if you did `ExpectCombinedOutput("foo \"bar bat\"", ...)` and then the program did `Command("foo", "\"bar bat\"")`, it would fail, with `Wrong Exec: expected []string{"foo", "\"bar", "bat\""}, got []string{"foo", "bar bat"}`, so it should at least be obvious to the developer that the quotes weren't being parsed the way they wanted.